src/components/v5/common/ActionSidebar/partials/motions/MotionSimplePayment/MotionSimplePayment.tsx

```diff
@@ -0,0 +1,75 @@
+import React, { FC, useMemo, useState } from 'react';
+import { formatText } from '~utils/intl';
+import Stepper from '~v5/shared/Stepper';
+import FinalizeStep from './steps/FinalizeStep';
+import OutcomeStep from './steps/OutcomeStep';
+import RevealStep from './steps/RevealStep';
+import StakingStep from './steps/StakingStep';
+import VotingStep from './steps/VotingStep';
+
+const MotionSimplePayment: FC = () => {
+  // @todo: pass current step to the state when other steps will be available
+  const [activeStepIndex, setActiveStepIndex] = useState(0);
+
+  const items = useMemo(
```

Comment:
maybe we will move this to utils file?

Reply:
A lot of logic may be added here, so I would leave it here for now. When this motion will be finished, then we can clean this up a bit

Reply:
Also, I'm using useMemo hook here, so it can't be moved to utils